packages/lesswrong/components/common/TabPicker.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useState} from 'react'
+import {Components, registerComponent} from '../../lib/vulcan-lib'
+import {AnalyticsContext, useTracking} from '../../lib/analyticsEvents.tsx'
+import classNames from 'classnames'
+
+const styles = (theme: ThemeType) => ({
+  tabsSection: {
+    marginTop: 10,
+    marginBottom: 26,
+    [theme.breakpoints.down('sm')]: {
+      marginTop: 20,
+    },
+  },
+  tabsRow: {
+    position: 'relative',
+  },
+  topicsBar: {
+    display: 'flex',
+    columnGap: 8,
+  },
+  tab: {
+    width: '150px',
+    backgroundColor: theme.palette.panelBackground.default,
+    color: theme.palette.grey[500],
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 14,
+    lineHeight: '23px',
+    fontWeight: '500',
+    padding: '4px 8px',
+    borderRadius: 3,
+    cursor: 'pointer',
+    '&:hover': {
+      color: theme.palette.grey[700],
+    },
+  },
+  activeTab: {
+    backgroundColor: theme.palette.grey[300],
+    color: theme.palette.grey[700],
+    '&:hover': {
+     color: theme.palette.grey[800],
+    },
+  },
+  tagDescriptionTooltip: {
+    margin: 8,
+  }
+})
+
+export interface TabRecord {
+  name: string,
+  label: string,
+  description?: string,
+}
+
+/**
+ * A horizontal bar of clickable tabs as alternative to a dropdown
+ * By default - displays core tags in order defined by defaultOrder. Used on EA and WakingUP home pages.
```

Comment:
Was this copied from somewhere?

Reply:
Overalll component is a stripped down version of TagHomeBar.  It seems I altered part of that comment but left the rest in.